DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Operations/Send/RootBaseObjectBuilder.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rhino.DocObjects;
+using Rhino;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Core.Models;
+using Speckle.Core.Kits;
+using System.Threading;
+
+namespace Speckle.Connectors.Rhino7.Operations.Send;
+
+/// <summary>
+/// Stateless builder object to turn an <see cref="ISendFilter"/> into a <see cref="Base"/> object
+/// </summary>
+public class RootBaseObjectBuilder
```

Comment:
I mean I know it returns a base, but I don't find it's descriptive enough as to what it is doing, is this the commit builder, maybe we drop Base and then it is RootObjectBuilder, that might be nicer...